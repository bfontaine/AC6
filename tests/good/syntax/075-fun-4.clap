val f = fun => 42

