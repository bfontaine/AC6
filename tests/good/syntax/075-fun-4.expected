val f =
  fun (_ : { U }) => 42
